lib/fs/MiqFS/modules/Ext4.rb

```diff
@@ -226,7 +226,7 @@ def ifs_getFile(p, miqfs = nil)
       dirObj = ifs_getDir(dir, miqfs)
       dirEnt = dirObj.nil? ? nil : dirObj.findEntry(fname)
     rescue RuntimeError => err
-      $log.error err.message.to_s if $log
+      $log.info err.message.to_s if $log
```

Comment:
What about other RuntimeErrors?  I think this should be more prescriptive either by catching a specific error or by inspecting the log message.